ci-operator/step-registry/lp-interop-tooling/archive-results/README.md

```diff
@@ -0,0 +1,35 @@
+# lp-interop-tooling-archive-results-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [~~Infrastructure~~](#infrastructure)
+  - [Variables](#variables)
+  - [~~Credentials~~](#credentials)
+  - [Other](#other)
+
+## Purpose
```

Comment:
I would like to better understand the scope/requirement for this new step. As mentioned it says its important for reporting process. Is this for the context of accessing results by another reporting application or just trying to reduce the number of directories a user needs to dig into the artifacts in order to find it?

I also see if its looking for an individual file to relocate, it is likely we would come across tests that run producing multiple XML files.

I just am trying to understand if this is a hard requirement to have or a nice to have. I would hate to see the job end up being flaky because this step starts failing.

Reply:
I wrote this anticipating it would be an important step. Let's discuss this in today's meeting.

Reply:
As we discussed in our meeting today, this step will really only be necessary if:

1. We decided to use Report Portal
    - The reason we would need it for Report Portal reporting is that the tool I have written needs to be able to reliably find the results file to report on, and this step offers an easy way to find it.. Currently, it is just reporting the Prow job results, but we would like to either report just the test results or a combination of Prow results and test results.

3. We want only the test results in TestGrid rather than the test results AND the Prow job results.
    -  We would probably need to update the filter in the TestGrid configurations to look for XML in this step's folder.